geotrek/common/urls.py

```diff
@@ -22,8 +27,19 @@ class LangConverter(converters.StringConverter):
     path('commands/syncview', sync_view, name='sync_randos_view'),
     path('commands/statesync/', sync_update_json, name='sync_randos_state'),
     path('api/<lang:lang>/themes.json', ThemeViewSet.as_view({'get': 'list'}), name="themes_json"),
+    path('hdviewpoint/add', HDViewPointCreate.as_view(), name="hdviewpoint_add"),
+    path('hdviewpoint/<int:pk>', HDViewPointDetail.as_view(), name="hdviewpoint_detail"),
+    path('hdviewpoint/edit/<int:pk>', HDViewPointUpdate.as_view(), name="hdviewpoint_change"),
+    path('hdviewpoint/delete/<int:pk>', HDViewPointDelete.as_view(), name="hdviewpoint_delete"),
+    path('hdviewpoint/annotate/<int:pk>', HDViewPointAnnotate.as_view(), name="hdviewpoint_annotate"),
 ]
 
+rest_router = DefaultRouter(trailing_slash=False)
+rest_router.register(r'api/' + HDViewPoint._meta.model_name + '/drf/' + HDViewPoint._meta.model_name + 's',
```

Comment:
as it's not like mapeneity tha tgenerate url from model names, juste set value here